packages/frontend-shared/src/gql-components/topnav/LoginModal.vue

```diff
@@ -72,10 +72,9 @@
             </div>
           </div>
         </DialogDescription>
-
         <div
-          v-if="showFooter"
           class="bg-gray-50 border-t-1px py-16px px-24px"
+          :class="{ 'hidden': !showFooter }"
```

Comment:
Outright un-mounting Auth here when an error occurs would trigger the reset logic, which would clear the error (not helpful). So I instead just hide the footer ('hidden' -> 'display: none;' in windicss).